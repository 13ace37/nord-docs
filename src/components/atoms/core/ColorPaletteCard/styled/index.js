/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Card from "./Card";
import CardWrapper from "./CardWrapper";
import Color from "./Color";
import ColorBox from "./ColorBox";
import FloatingTitle from "./FloatingTitle";
import Label from "./Label";
import Palette from "./Palette";
import Title from "./Title";

export { Card, CardWrapper, Color, ColorBox, FloatingTitle, Label, Palette, Title };
