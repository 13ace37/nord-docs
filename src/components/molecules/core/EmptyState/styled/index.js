/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Illustration, { illustrationVariants } from "./Illustration";
import Headline from "./Headline";
import Subline from "./Subline";
import Wrapper from "./Wrapper";
import TextWrapper from "./TextWrapper";

export { Headline, Illustration, Subline, TextWrapper, Wrapper, illustrationVariants };
