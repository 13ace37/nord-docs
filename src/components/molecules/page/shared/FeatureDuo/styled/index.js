/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Actions from "./Actions";
import Headline from "./Headline";
import Subline from "./Subline";
import Text from "./Text";
import Visualization from "./Visualization";
import Wrapper from "./Wrapper";

export { Actions, Headline, Subline, Text, Visualization, Wrapper };
