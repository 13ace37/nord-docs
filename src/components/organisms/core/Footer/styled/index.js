/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Category from "./Category";
import Copyright from "./Copyright";
import Grid from "./Grid";
import Metadata from "./Metadata";
import OrganizationBrand from "./OrganizationBrand";
import OrganizationBrandCaption from "./OrganizationBrandCaption";
import Sitemap from "./Sitemap";
import SitemapCategory from "./SitemapCategory";
import SitemapLink from "./SitemapLink";
import SitemapList from "./SitemapList";
import SocialNetworking from "./SocialNetworking";

export {
  Category,
  Copyright,
  Grid,
  Metadata,
  OrganizationBrand,
  OrganizationBrandCaption,
  Sitemap,
  SitemapCategory,
  SitemapLink,
  SitemapList,
  SocialNetworking
};
