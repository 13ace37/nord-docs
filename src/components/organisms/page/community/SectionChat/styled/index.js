/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import ChatCard from "./ChatCard/ChatCard";
import Grid from "./Grid";

export { ChatCard, Grid };
