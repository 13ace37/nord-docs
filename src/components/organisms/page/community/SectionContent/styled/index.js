/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Grid from "./Grid";
import RedditCard from "./RedditCard/RedditCard";
import StackOverflowCard from "./StackOverflowCard/StackOverflowCard";
import WaveDivider from "./WaveDivider";

export { Grid, RedditCard, StackOverflowCard, WaveDivider };
