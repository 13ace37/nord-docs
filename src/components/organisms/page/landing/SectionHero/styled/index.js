/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

/**
 * @file Provides shared components for the landing page orgaism components.
 * @author Arctic Ice Studio <development@arcticicestudio.com>
 * @author Sven Greb <development@svengreb.de>
 * @since 0.6.0
 */

import Cloud1 from "./Cloud1";
import Cloud2 from "./Cloud2";
import Cloud3 from "./Cloud3";
import DecorationCanvas from "./DecorationCanvas";
import WaveDivider from "./WaveDivider";

export { Cloud1, Cloud2, Cloud3, DecorationCanvas, WaveDivider };
