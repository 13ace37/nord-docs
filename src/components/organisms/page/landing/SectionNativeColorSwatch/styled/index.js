/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import DecorationCanvas from "./DecorationCanvas";
import SketchTools from "./SketchTools";
import VectorDesignCollage from "./VectorDesignCollage";
import WaveDivider from "./WaveDivider";

export { DecorationCanvas, SketchTools, VectorDesignCollage, WaveDivider };
