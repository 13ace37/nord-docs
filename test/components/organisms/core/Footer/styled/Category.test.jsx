/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import React from "react";

import { renderWithTheme } from "nord-docs-test-utils";
import { Category } from "organisms/core/Footer/styled";

describe("theme styles", () => {
  test("has visible border-bottom when matching the active route", () => {
    const { container } = renderWithTheme(<Category isActiveRouteMatch>Nord</Category>);

    expect(container.firstChild).toHaveStyleRule("border-bottom", expect.not.stringContaining("transparent"));
    expect(container.firstChild).toMatchSnapshot();
  });

  test("has invisible border-bottom when not matching the active route", () => {
    const { container } = renderWithTheme(<Category>Nord</Category>);

    expect(container.firstChild).toHaveStyleRule("border-bottom", expect.stringContaining("transparent"));
    expect(container.firstChild).toMatchSnapshot();
  });
});
