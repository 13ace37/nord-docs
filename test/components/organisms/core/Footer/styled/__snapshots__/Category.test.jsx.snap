// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme styles has invisible border-bottom when not matching the active route 1`] = `
.c0 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  font-size: 1.125em;
  color: #7b88a1;
  border-bottom: 2px solid transparent;
  -webkit-transition: border-bottom 400ms ease-in-out, color 400ms ease-in-out;
  transition: border-bottom 400ms ease-in-out, color 400ms ease-in-out;
}

<div
  class="c0"
>
  Nord
</div>
`;

exports[`theme styles has visible border-bottom when matching the active route 1`] = `
.c0 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  font-size: 1.125em;
  color: #4c566a;
  border-bottom: 2px solid #4c566a;
  -webkit-transition: border-bottom 400ms ease-in-out, color 400ms ease-in-out;
  transition: border-bottom 400ms ease-in-out, color 400ms ease-in-out;
}

<div
  class="c0"
>
  Nord
</div>
`;
